src/routes/tags/create-tag.e2e.test.js

```diff
@@ -0,0 +1,139 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const { fixtures } = require('../../../test/utils')
+
+describe('POST /tags', () => {
+
+	let user
+	let accessToken
+
+	before(async ()=> {
+
+		user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+
+	})
+
+	describe('Failure', () => {
```

Comment:
kindly add a test case for when the tag might already exist

Reply:
yes please